src/eq_schema/schema/Question/index.js

```diff
@@ -31,19 +31,46 @@ const findMutualOption = flow(
 
 const findMutuallyExclusive = flow(get("answers"), find(findMutualOption));
 
-const processPipe = (ctx) => flow(convertPipes(ctx), getInnerHTMLWithPiping);
+const processPipe = (ctx, isMultipleChoiceValue = false, isRepeatingSection) =>
+  flow(
+    convertPipes(ctx, isMultipleChoiceValue, isRepeatingSection),
+    getInnerHTMLWithPiping
+  );
 const reversePipe = (ctx) =>
   flow(wrapContents("contents"), reversePipeContent(ctx));
 
+const getSectionByPageId = (ctx, pageId) => {
```

Comment:
Can we put this function into a new module calleed sectionGetters.js in the utils>functions folder. that way it's available everywhere.